laske_export/document/invoice_sales_order_adapter.py

```diff
@@ -304,5 +319,133 @@ def set_values(self):
         self.set_dates()
         self.set_references()
 
-        line_items = self.get_line_items()
-        self.sales_order.line_items = line_items
+        self.sales_order.line_items = self.get_line_items()
+
+
+class AkvInvoiceSalesOrderAdapter(InvoiceSalesOrderAdapter):
+    """Adapter for service unit Alueiden kehittäminen ja valvonta.
+
+    For service-unit-aware instantiation of class objects, please use the
+    factory function invoice_sales_order_adapter_factory
+    """
+
+    AKV_DATE_FORMAT = "%d.%m.%Y"
+
+    def get_bill_text(self) -> str:
+        """Create bill text
+
+        AKV requested that "Hki otsikko ulkoinen" is left empty in their SAP.
+        "Hki otsikko ulkoinen" corresponds to XML elements BillTextL<number> in
+        the MVJ export.
+        """
+        number_of_bill_text_lines = 6
```

Comment:
variable name could be capitalized

Reply:
I don't see a need: the variable is spelled out only to explain the number 6 and is used immediately on the next line. If it was declared on top of the module or in a very long function, I would do the capitalization. In this case I think there is no need to distinguish between function-local and module level variables with the capitalization.

Reply:
... or the variable could be moved to be a class property on the superclass.

Reply:
Lets leave it as is, that makes sense too